client/src/components/OverviewComponent/index.js

```diff
@@ -17,11 +16,12 @@ const OverviewComponent = ({
   isLeftImg,
   isHomePage,
   isDark,
+  isRed,
 }) => {
   return (
     <div className="overview-container">
       {isLeftImg && (
-        <div className="container-photo">
+        <div className={isRed ? 'redGround' : 'whiteGround'}>
           <img className="home-photo" src={imageSource} alt="Home" />
```

Comment:
I suggest that class name to be descriptive, because this component used more than one time
for example 
```suggestion
          <img className="background-photo" src={imageSource} alt="Home" />
```